homeassistant/components/telegram_bot/config_flow.py

```diff
@@ -178,7 +180,44 @@ def async_get_supported_subentry_types(
     # triggered by async_setup() from __init__.py
     async def async_step_import(self, import_data: dict[str, str]) -> ConfigFlowResult:
         """Handle import of config entry from configuration.yaml."""
-        return await self.async_step_user(import_data)
+        config_flow_result: ConfigFlowResult = await self.async_step_user(import_data)
+
+        if config_flow_result["type"] == FlowResultType.FORM:
+            async_create_issue(
+                self.hass,
+                DOMAIN,
+                "deprecated_yaml_import_issue_unknown",
+                breaks_in_ha_version="2025.9.0",
```

Comment:
```suggestion
                breaks_in_ha_version="2025.12.0",
```

Reply:
We require 6 months of deprecation time from the release that deprecated the YAML configuration.